kit.yml

```diff
@@ -313,7 +314,7 @@ certificates:
         usage:     [ client_auth ]
 
   self-signed:
-    haproxy:
+    haproxy/certs:
```

Comment:
why this?

Reply:
To comply with our implicit contract that all generated certs go under <caID>/certs/<certname>

Reply:
It should be a no-op for the clients as we move existing certs for them in check hook